data/releases/schedule.yaml

```diff
@@ -53,11 +65,14 @@ schedules:
       note: Regression https://groups.google.com/g/kubernetes-dev/c/KuF8s2zueFs
 - release: 1.20
   releaseDate: 2020-12-08
-  next: 1.20.14
-  cherryPickDeadline: 2021-12-10
-  targetDate: 2021-12-15
+  next: 1.20.15
```

Comment:
Do we want to add another 1.20 patch release to the calendar? If I'm not mistaken, 1.20 is now in the maintenance mode, so we should be doing patch releases for it only if it's needed (e.g. security fixes, critical bugs, etc.)?

cc @kubernetes/release-engineering

Reply:
we can schedule that and when is more closer to the day we decide